Library/ViewModels/RewardCardContainerViewModel.swift

```diff
@@ -76,16 +76,6 @@ public final class RewardCardContainerViewModel: RewardCardContainerViewModelTyp
     self.rewardSelected = reward
       .takeWhen(self.pledgeButtonTappedProperty.signal)
       .map { $0.id }
-
-    // Tracking
-    projectAndReward
```

Comment:
Moved this to `RewardsCollectionViewModel`, which has access to the `context` and the `refTag`.